### DISCUSSION TITLE
Change description of package

### DISCUSSION THREAD

**Discussion by Tklaversma** [spatie/holidays#123] [Q&A]
Not every holiday is equal to having free from work... at least not in The Netherlands. 

I would change the description of the package since this is a holiday package, and not an "free from work package". 

Agree @Nielsvanpach ?

**Comment by jpgnz** [comment]
I was going to ask the same thing, In New Zealand:

> New Year's Day: 1 January 	
> Day after New Year's Day: 2 January 	
> Christmas Day: 25 December 	
> Boxing Day: 26 December
> 
> Christmas Day, Boxing Day, New Year's Day and 2 January holiday: These public holidays are observed on the actual day when they fall on a weekday. When they fall on a Saturday/Sunday:
> 
> - If the employee would normally have worked on the Saturday/Sunday, the public holiday is observed on the Saturday/Sunday.
> - If the employee would not normally have worked on the Saturday/Sunday, the public holiday is observed on the following Monday.
> 
> In the case that New Years Day is the Sunday, Day after New Year's Day would be the Monday, so the Public Holiday for those who would have normally worked Monday and Tuesday, New Years Day is observed on the Tuesday.

We would call that the difference between Actual date vs Observed date. I am looking at this package as Actual Date currently.

**Reply by Tklaversma** [reply]
For the Netherlands it's a bit different. For example, "Bevrijdingsdag" is a official holiday, but employers decide whether the employees are free from work or not. Most employers say "once every 5 years", others say "it's just a normal work day" and some say "your free from work".  Besides this particular holiday I'm describing, there are more of these cases.

Even the official government website states that holidays do not mean your employer is obligated to give you free from work, see here https://www.rijksoverheid.nl/onderwerpen/arbeidsovereenkomst-en-cao/vraag-en-antwoord/officiele-feestdagen (in Dutch). 

But the cases we're making are the same, holidays !== free from work, so the description of this package should change.
